opencood/hypes_yaml/point_pillar_fcooper.yaml

```diff
@@ -81,7 +81,7 @@ model:
       upsample_strides: [1, 2, 4]
       num_upsample_filter: [128, 128, 128]
     shrink_header:
-      kernal_size: [ 1 ]
```

Comment:
I know there is a typo here. But all the pretrained models yaml files are using this, it would be a little complicated to modify all of those pretrain models' yaml file. Would you mind change back to the 'kernal_size'? Thanks!

Reply:
Sure, sure. No problem.